manifests/setup/kustomization.yaml

```diff
@@ -1,12 +1,7 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
-namespace: kubesphere-logging-system
-commonLabels:
-  app.kubernetes.io/name: fluentbit-operator
-
 resources:
-- namespace-namespace.yaml
 - fluentbit-operator-clusterRoleBinding.yaml
 - fluentbit-operator-deployment.yaml
```

Comment:
There is nothing kustomized in the new commit now, do we still need this？

Reply:
Yes, this is still needed. All adjustments happen downstream, but require a kustomize file upstream, e.g.

```
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization

resources:
  - github.com/sighupio/gatekeeper-policy-manager?ref=v0.4.1
  - ingress.yaml

patchesStrategicMerge:
  - set-memory.yaml
```